src/protocol.h

```diff
@@ -71,6 +71,8 @@
 #define PROTMESSID_CLM_REQ_VERSION_AND_OS     1012 // request version number and operating system
 #define PROTMESSID_CLM_CONN_CLIENTS_LIST      1013 // channel infos for connected clients
 #define PROTMESSID_CLM_REQ_CONN_CLIENTS_LIST  1014 // request the connected clients list
+#define PROTMESSID_CLM_REQ_CHANNEL_LEVEL_LIST 1015 // request the channel level list
+#define PROTMESSID_CLM_CHANNEL_LEVEL_LIST     1016 // channel level list
```

Comment:
As I wrote above: For this message (updating the levels) it is perfect to use a connection-less message (just to make sure for that for this message type no change is required).